src/infer_response.h

```diff
@@ -255,6 +265,14 @@ class InferenceResponse {
   // The response outputs.
   const std::deque<Output>& Outputs() const { return outputs_; }
 
+  // The response index.
+  uint64_t ResponseIdx() const { return response_idx_; }
+
+  // The unique request ID stored in triton.
+  uint64_t RequestUniqueId() const { return request_id_; }
+
+  uint64_t ResponseStartNs() const { return response_start_; }
```

Comment:
Add a descriptive comment here for uniformity.

Reply:
Updated.